src/validation.cpp

```diff
@@ -5237,3 +5237,33 @@ void ChainstateManager::Reset()
     m_active_chainstate = nullptr;
     m_snapshot_validated = false;
 }
+
+void ChainstateManager::MaybeRebalanceCaches()
```

Comment:
In commit "add ChainstateManager::MaybeRebalanceCaches()" (69514bd9fce0f867256bd08ec9f1b616a3ece573)

Could probably write a short unit test exercising this (checking m_coins{tip,db}_cache_size_bytes)

Reply:
Good idea, done.